R/document.r

```diff
@@ -20,20 +20,24 @@ document <- function(pkg = ".", roclets = NULL) {
     rstudioapi::documentSaveAll()
   }
 
-  # Refresh the pkg structure with any updates to the Collate entry
-  # in the DESCRIPTION file
-  roxygen2::update_collate(pkg$path)
+  if (packageVersion("roxygen2") >= "6.1.0") {
+    withr::with_envvar(r_env_vars(), roxygen2::roxygenise(pkg$path, roclets))
+  } else {
+    # Refresh the pkg structure with any updates to the Collate entry
+    # in the DESCRIPTION file
+    roxygen2::update_collate(pkg$path)
 
-  roclets <- roclets %||% roxygen2::load_options(pkg$path)$roclets
-  roclets <- setdiff(roclets, "collate")
+    roclets <- roclets %||% roxygen2::load_options(pkg$path)$roclets
+    roclets <- setdiff(roclets, "collate")
 
-  if (packageVersion("roxygen2") < "6.1.0") {
-    load_all(pkg$path, helpers = FALSE)
-  }
+    if (packageVersion("roxygen2") < "6.1.0") {
```

Comment:
We can remove this conditional I guess right, the only way you could be in this block is by being less than "6.1.0"

Reply:
Doh